libcfnet/misc.c

```diff
@@ -53,41 +53,6 @@ int cf_closesocket(int sd)
     return res;
 }
 
-/* Convert IP address in src (which can be struct sockaddr_storage (best
- * choice for any IP version), struct sockaddr, struct sockaddr_in or struct
- * sockaddr_in6) to string dst.
- * Better use getnameinfo(NI_NUMERICHOST) if available. */
-const char *sockaddr_ntop(const void *src, char *dst, socklen_t size)
```

Comment:
Yes, but might be false positive. I did not bother to check rather remove it as it was unused.